app/actions/app/global.ts

```diff
@@ -39,3 +52,44 @@ export const storeProfileLongPressTutorial = async (prepareRecordsOnly = false)
         return {error};
     }
 };
+
+export async function resetAfterCRTChange(serverUrl: string): Promise<{error?: any; userId?: string}> {
+    const operator = DatabaseManager.serverDatabases?.[serverUrl].operator;
+
+    if (!operator) {
+        return {error: 'No App database found'};
+    }
+
+    const database = operator.database;
+
+    try {
+        await database.write(() => {
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            return database.adapter.unsafeExecute({
+                sqls: [
+                    [`DELETE FROM ${POST}`, []],
+                    [`DELETE FROM ${POSTS_IN_CHANNEL}`, []],
+                    [`DELETE FROM ${POSTS_IN_THREAD}`, []],
+                    [`DELETE FROM ${THREAD}`, []],
+                    [`DELETE FROM ${THREADS_IN_TEAM}`, []],
+                    [`DELETE FROM ${THREAD_PARTICIPANT}`, []],
+                    [`DELETE FROM ${MY_CHANNEL}`, []],
+                ],
+            });
+        });
+
+        const currentServerUrl = await DatabaseManager.getActiveServerUrl();
+        await resetWebSocketLastDisconnected(operator);
+        if (currentServerUrl === serverUrl) {
+            popToRoot();
+        }
+
+        return appEntry(serverUrl);
```

Comment:
@enahum since we are deleting all data from those tables, shouldn't we fetch data since 0?

Reply:
If it's safe to change it to `Date.now` then great. I'm not sure how to test it.

Reply:
I thi k you are going to need a new optional parameter, casuse 0 will execute what I mentioned, but since will prevent certain channels from being fetch and you need that data